### ISSUE TITLE
bug: Creating SQS with aws cli fails with "Unable to parse response" error

### ISSUE THREAD

**Issue by ramakot** [localstack/localstack#4777]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I am able to start localstack. while creating Queue I am getting the following error 

```
aws --endpoint-url=http://localhost:4566 sqs create-queue --queue-name test-queue

Unable to parse response (not well-formed (invalid token): line 1, column 0), invalid XML received. Further retries may succeed:
b'{}'
```

Localstack logs:
```
WARNING:localstack.utils.server.http2_server: Error in proxy handler for request POST http://localhost:4566/: 'NoneType' object has no attribute 'group' Traceback (most recent call last):
  File "/opt/code/localstack/localstack/utils/server/http2_server.py", line 159, in index
    result = await run_sync(handler, request, data)
  File "/opt/code/localstack/localstack/utils/async_utils.py", line 86, in run_sync
    return await loop.run_in_executor(thread_pool, copy_context().run, func_wrapped)
  File "/usr/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 545, in handler
    server_address=parsed_url.netloc,
  File "/opt/code/localstack/localstack/utils/aws/request_context.py", line 206, in modify_and_forward
    method, path, data_bytes=data_bytes, headers=headers, *args, **kwargs
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 330, in modify_and_forward
    method=method, path=path, data=data, headers=headers
  File "/opt/code/localstack/localstack/services/edge.py", line 179, in forward_request
    result = do_forward_request(api, method, path, data, headers, port=port)
  File "/opt/code/localstack/localstack/services/edge.py", line 230, in do_forward_request
    result = do_forward_request_inmem(api, method, path, data, headers, port=port)
  File "/opt/code/localstack/localstack/services/edge.py", line 262, in do_forward_request_inmem
    server_address=server_address,
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 405, in modify_and_forward
    updated_response = update_listener.return_response(**kwargs)
  File "/opt/code/localstack/localstack/services/sqs/sqs_listener.py", line 439, in return_response
    queue_url = re.match(regex, content_str, re.DOTALL).group(1)
AttributeError: 'NoneType' object has no attribute 'group'
```

### Expected Behavior

Should be supposed SQS create queue

### How are you starting LocalStack?

With the `localstack` script

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker run localstack/localstack

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

    awslocal s3 mb s3://mybucket

### Environment

```markdown
- OS: MacOS Bigsur
- LocalStack: 0.12.18.5
```

### Anything else?

```
WARNING:localstack.utils.server.http2_server: Error in proxy handler for request POST http://localhost:4566/: 'NoneType' object has no attribute 'group' Traceback (most recent call last):
  File "/opt/code/localstack/localstack/utils/server/http2_server.py", line 159, in index
    result = await run_sync(handler, request, data)
  File "/opt/code/localstack/localstack/utils/async_utils.py", line 86, in run_sync
    return await loop.run_in_executor(thread_pool, copy_context().run, func_wrapped)
  File "/usr/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 545, in handler
    server_address=parsed_url.netloc,
  File "/opt/code/localstack/localstack/utils/aws/request_context.py", line 206, in modify_and_forward
    method, path, data_bytes=data_bytes, headers=headers, *args, **kwargs
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 330, in modify_and_forward
    method=method, path=path, data=data, headers=headers
  File "/opt/code/localstack/localstack/services/edge.py", line 179, in forward_request
    result = do_forward_request(api, method, path, data, headers, port=port)
  File "/opt/code/localstack/localstack/services/edge.py", line 230, in do_forward_request
    result = do_forward_request_inmem(api, method, path, data, headers, port=port)
  File "/opt/code/localstack/localstack/services/edge.py", line 262, in do_forward_request_inmem
    server_address=server_address,
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 405, in modify_and_forward
    updated_response = update_listener.return_response(**kwargs)
  File "/opt/code/localstack/localstack/services/sqs/sqs_listener.py", line 439, in return_response
    queue_url = re.match(regex, content_str, re.DOTALL).group(1)
AttributeError: 'NoneType' object has no attribute 'group'
```

**Comment by macnev2013** [comment]
Thanks for reporting the bug @ramakot , I tried with the latest docker image - Not able to reproduce the issue. Can you please try with the latest docker image? We would be happy to get it fixed if the issue still persists. Thanks.

**Comment by ramakot** [comment]
I have downloaded the latest docker and still issue exist. I am able to create S3 bucket But unable to create queue. when I try to execute the queue through aws cli (aws --endpoint-url=http://localhost:4566 sqs create-queue --queue-name test_queue). it's giving me localstack error is File "/opt/code/localstack/localstack/utils/server/http2_server.py", line 159, in index
    result = await run_sync(handler, request, data)
  File "/opt/code/localstack/localstack/utils/async_utils.py", line 86, in run_sync
    return await loop.run_in_executor(thread_pool, copy_context().run, func_wrapped)
  File "/usr/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 544, in handler
    server_address=parsed_url.netloc,
  File "/opt/code/localstack/localstack/utils/aws/request_context.py", line 206, in modify_and_forward
    method, path, data_bytes=data_bytes, headers=headers, *args, **kwargs
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 329, in modify_and_forward
    method=method, path=path, data=data, headers=headers
  File "/opt/code/localstack/localstack/services/edge.py", line 182, in forward_request
    result = do_forward_request(api, method, path, data, headers, port=port)
  File "/opt/code/localstack/localstack/services/edge.py", line 237, in do_forward_request
    result = do_forward_request_inmem(api, method, path, data, headers, port=port)
  File "/opt/code/localstack/localstack/services/edge.py", line 269, in do_forward_request_inmem
    server_address=server_address,
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 404, in modify_and_forward
    updated_response = update_listener.return_response(**kwargs)
  File "/opt/code/localstack/localstack/services/sqs/sqs_listener.py", line 439, in return_response
    queue_url = re.match(regex, content_str, re.DOTALL).group(1)
AttributeError: 'NoneType' object has no attribute 'group'

**Comment by thrau** [comment]
hi @ramakot , please help us further investigate the cause for the issue:

* could you please share with us the aws cli version you are using? `aws --version`
* are you having the same problem when using `awslocal` instead of `aws?
* please run the aws command with the `--debug` flag, so `aws --debug --endpoint-url=http://localhost:4566 sqs create-queue --queue-name test-queue`

**Comment by ramakot** [comment]
1. could you please share with us the aws cli version you are using? aws --version
Ans: aws-cli/2.2.7 Python/3.8.8 Darwin/20.6.0 exe/x86_64 prompt/off
are you having the same problem when using awslocal instead of aws?
Ans: I have tried with awslocal. It's same problem
please run the aws command with the --debug flag, so aws --debug --endpoint-url=http://localhost:4566 sqs create-queue --queue-name test-queue and share the entire output
Ans: rkottamasu@USCHIRKOTTAMA1 localstack % aws --debug --endpoint-url=http://localhost:4566 sqs create-queue --queue-name test-queue     
2021-11-15 11:54:05,332 - MainThread - awscli.clidriver - DEBUG - CLI version: aws-cli/2.2.7 Python/3.8.8 Darwin/20.6.0 exe/x86_64
2021-11-15 11:54:05,332 - MainThread - awscli.clidriver - DEBUG - Arguments entered to CLI: ['--debug', '--endpoint-url=http://localhost:4566', 'sqs', 'create-queue', '--queue-name', 'test-queue']
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <function add_s3 at 0x7fbcd4df7940>
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <function add_ddb at 0x7fbcd4c39040>
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <bound method BasicCommand.add_command of <class 'awscli.customizations.configure.configure.ConfigureCommand'>>
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <function change_name at 0x7fbcd4bdd790>
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <function change_name at 0x7fbcd4be4790>
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <function alias_opsworks_cm at 0x7fbcd4e093a0>
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <function add_history_commands at 0x7fbcd4c81dc0>
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <bound method BasicCommand.add_command of <class 'awscli.customizations.devcommands.CLIDevCommand'>>
2021-11-15 11:54:05,360 - MainThread - botocore.hooks - DEBUG - Event building-command-table.main: calling handler <function add_waiters at 0x7fbcd4e005e0>
2021-11-15 11:54:05,360 - MainThread - botocore.loaders - DEBUG - Loading JSON file: /usr/local/aws-cli/awscli/data/cli.json
2021-11-15 11:54:05,363 - MainThread - botocore.hooks - DEBUG - Event top-level-args-parsed: calling handler <function resolve_types at 0x7fbcd4d328b0>
2021-11-15 11:54:05,363 - MainThread - botocore.hooks - DEBUG - Event top-level-args-parsed: calling handler <function no_sign_request at 0x7fbcd4d34430>
2021-11-15 11:54:05,363 - MainThread - botocore.hooks - DEBUG - Event top-level-args-parsed: calling handler <function resolve_verify_ssl at 0x7fbcd4d343a0>
2021-11-15 11:54:05,363 - MainThread - botocore.hooks - DEBUG - Event top-level-args-parsed: calling handler <function resolve_cli_read_timeout at 0x7fbcd4d34550>
2021-11-15 11:54:05,363 - MainThread - botocore.hooks - DEBUG - Event top-level-args-parsed: calling handler <function resolve_cli_connect_timeout at 0x7fbcd4d344c0>
2021-11-15 11:54:05,363 - MainThread - botocore.hooks - DEBUG - Event top-level-args-parsed: calling handler <built-in method update of dict object at 0x7fbcd4ea1500>
2021-11-15 11:54:05,363 - MainThread - awscli.clidriver - DEBUG - CLI version: aws-cli/2.2.7 Python/3.8.8 Darwin/20.6.0 exe/x86_64 prompt/off
2021-11-15 11:54:05,363 - MainThread - awscli.clidriver - DEBUG - Arguments entered to CLI: ['--debug', '--endpoint-url=http://localhost:4566', 'sqs', 'create-queue', '--queue-name', 'test-queue']
2021-11-15 11:54:05,363 - MainThread - botocore.hooks - DEBUG - Event session-initialized: calling handler <function add_timestamp_parser at 0x7fbcd4df7f70>
2021-11-15 11:54:05,363 - MainThread - botocore.hooks - DEBUG - Event session-initialized: calling handler <function register_uri_param_handler at 0x7fbcd498fdc0>
2021-11-15 11:54:05,364 - MainThread - botocore.hooks - DEBUG - Event session-initialized: calling handler <function add_binary_formatter at 0x7fbcd4e66a60>
2021-11-15 11:54:05,364 - MainThread - botocore.hooks - DEBUG - Event session-initialized: calling handler <function no_pager_handler at 0x7fbcd498d280>
2021-11-15 11:54:05,364 - MainThread - botocore.hooks - DEBUG - Event session-initialized: calling handler <function inject_assume_role_provider_cache at 0x7fbcd49ee790>
2021-11-15 11:54:05,366 - MainThread - botocore.utils - DEBUG - IMDS ENDPOINT: http://169.254.169.254/
2021-11-15 11:54:05,372 - MainThread - botocore.hooks - DEBUG - Event session-initialized: calling handler <function attach_history_handler at 0x7fbcd4c81ca0>
2021-11-15 11:54:05,372 - MainThread - botocore.hooks - DEBUG - Event session-initialized: calling handler <function inject_json_file_cache at 0x7fbcd4c36ee0>
2021-11-15 11:54:05,400 - MainThread - botocore.loaders - DEBUG - Loading JSON file: /usr/local/aws-cli/botocore/data/sqs/2012-11-05/service-2.json
2021-11-15 11:54:05,403 - MainThread - botocore.hooks - DEBUG - Event building-command-table.sqs: calling handler <function add_waiters at 0x7fbcd4e005e0>
2021-11-15 11:54:05,434 - MainThread - awscli.clidriver - DEBUG - OrderedDict([('queue-name', <awscli.arguments.CLIArgument object at 0x7fbcd61d6760>), ('attributes', <awscli.arguments.CLIArgument object at 0x7fbcd61d6790>), ('tags', <awscli.arguments.CLIArgument object at 0x7fbcd61d67c0>)])
2021-11-15 11:54:05,434 - MainThread - botocore.hooks - DEBUG - Event building-argument-table.sqs.create-queue: calling handler <function add_streaming_output_arg at 0x7fbcd4dfb550>
2021-11-15 11:54:05,434 - MainThread - botocore.hooks - DEBUG - Event building-argument-table.sqs.create-queue: calling handler <function add_cli_input_json at 0x7fbcd49f3040>
2021-11-15 11:54:05,435 - MainThread - botocore.hooks - DEBUG - Event building-argument-table.sqs.create-queue: calling handler <function add_cli_input_yaml at 0x7fbcd49f3310>
2021-11-15 11:54:05,435 - MainThread - botocore.hooks - DEBUG - Event building-argument-table.sqs.create-queue: calling handler <function unify_paging_params at 0x7fbcd4c39670>
2021-11-15 11:54:05,469 - MainThread - botocore.loaders - DEBUG - Loading JSON file: /usr/local/aws-cli/botocore/data/sqs/2012-11-05/paginators-1.json
2021-11-15 11:54:05,469 - MainThread - botocore.hooks - DEBUG - Event building-argument-table.sqs.create-queue: calling handler <function add_generate_skeleton at 0x7fbcd4d26e50>
2021-11-15 11:54:05,470 - MainThread - botocore.hooks - DEBUG - Event before-building-argument-table-parser.sqs.create-queue: calling handler <bound method OverrideRequiredArgsArgument.override_required_args of <awscli.customizations.cliinput.CliInputJSONArgument object at 0x7fbcd61d6850>>
2021-11-15 11:54:05,470 - MainThread - botocore.hooks - DEBUG - Event before-building-argument-table-parser.sqs.create-queue: calling handler <bound method OverrideRequiredArgsArgument.override_required_args of <awscli.customizations.cliinput.CliInputYAMLArgument object at 0x7fbcd61d68e0>>
2021-11-15 11:54:05,470 - MainThread - botocore.hooks - DEBUG - Event before-building-argument-table-parser.sqs.create-queue: calling handler <bound method GenerateCliSkeletonArgument.override_required_args of <awscli.customizations.generatecliskeleton.GenerateCliSkeletonArgument object at 0x7fbcd61d69d0>>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event load-cli-arg.sqs.create-queue.queue-name: calling handler <awscli.paramfile.URIArgumentHandler object at 0x7fbcd611e6a0>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event process-cli-arg.sqs.create-queue: calling handler <awscli.argprocess.ParamShorthandParser object at 0x7fbcd49b3af0>
2021-11-15 11:54:05,471 - MainThread - awscli.arguments - DEBUG - Unpacked value of 'test-queue' for parameter "queue_name": 'test-queue'
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event load-cli-arg.sqs.create-queue.attributes: calling handler <awscli.paramfile.URIArgumentHandler object at 0x7fbcd611e6a0>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event load-cli-arg.sqs.create-queue.tags: calling handler <awscli.paramfile.URIArgumentHandler object at 0x7fbcd611e6a0>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event load-cli-arg.sqs.create-queue.cli-input-json: calling handler <awscli.paramfile.URIArgumentHandler object at 0x7fbcd611e6a0>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event load-cli-arg.sqs.create-queue.cli-input-yaml: calling handler <awscli.paramfile.URIArgumentHandler object at 0x7fbcd611e6a0>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event load-cli-arg.sqs.create-queue.generate-cli-skeleton: calling handler <awscli.paramfile.URIArgumentHandler object at 0x7fbcd611e6a0>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event calling-command.sqs.create-queue: calling handler <bound method CliInputArgument.add_to_call_parameters of <awscli.customizations.cliinput.CliInputJSONArgument object at 0x7fbcd61d6850>>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event calling-command.sqs.create-queue: calling handler <bound method CliInputArgument.add_to_call_parameters of <awscli.customizations.cliinput.CliInputYAMLArgument object at 0x7fbcd61d68e0>>
2021-11-15 11:54:05,471 - MainThread - botocore.hooks - DEBUG - Event calling-command.sqs.create-queue: calling handler <bound method GenerateCliSkeletonArgument.generate_skeleton of <awscli.customizations.generatecliskeleton.GenerateCliSkeletonArgument object at 0x7fbcd61d69d0>>
2021-11-15 11:54:05,472 - MainThread - botocore.credentials - DEBUG - Looking for credentials via: env
2021-11-15 11:54:05,472 - MainThread - botocore.credentials - DEBUG - Looking for credentials via: assume-role
2021-11-15 11:54:05,472 - MainThread - botocore.credentials - DEBUG - Looking for credentials via: assume-role-with-web-identity
2021-11-15 11:54:05,472 - MainThread - botocore.credentials - DEBUG - Looking for credentials via: sso
2021-11-15 11:54:05,472 - MainThread - botocore.credentials - DEBUG - Looking for credentials via: shared-credentials-file
2021-11-15 11:54:05,472 - MainThread - botocore.credentials - INFO - Found credentials in shared credentials file: ~/.aws/credentials
2021-11-15 11:54:05,473 - MainThread - botocore.loaders - DEBUG - Loading JSON file: /usr/local/aws-cli/botocore/data/endpoints.json
2021-11-15 11:54:05,481 - MainThread - botocore.hooks - DEBUG - Event choose-service-name: calling handler <function handle_service_name_alias at 0x7fbcd40c4ee0>
2021-11-15 11:54:05,482 - MainThread - botocore.hooks - DEBUG - Event creating-client-class.sqs: calling handler <function add_generate_presigned_url at 0x7fbcd40741f0>
2021-11-15 11:54:05,483 - MainThread - botocore.regions - DEBUG - Creating a regex based endpoint for sqs, us-west2
2021-11-15 11:54:05,485 - MainThread - botocore.endpoint - DEBUG - Setting sqs timeout as (60, 60)
2021-11-15 11:54:05,486 - MainThread - botocore.hooks - DEBUG - Event provide-client-params.sqs.CreateQueue: calling handler <function base64_decode_input_blobs at 0x7fbcd4e651f0>
2021-11-15 11:54:05,486 - MainThread - botocore.hooks - DEBUG - Event before-parameter-build.sqs.CreateQueue: calling handler <function generate_idempotent_uuid at 0x7fbcd40e3ee0>
2021-11-15 11:54:05,486 - MainThread - botocore.hooks - DEBUG - Event before-call.sqs.CreateQueue: calling handler <function inject_api_version_header_if_needed at 0x7fbcd40ea790>
2021-11-15 11:54:05,486 - MainThread - botocore.endpoint - DEBUG - Making request for OperationModel(name=CreateQueue) with params: {'url_path': '/', 'query_string': '', 'method': 'POST', 'headers': {'Content-Type': 'application/x-www-form-urlencoded; charset=utf-8', 'User-Agent': 'aws-cli/2.2.7 Python/3.8.8 Darwin/20.6.0 exe/x86_64 prompt/off command/sqs.create-queue'}, 'body': {'Action': 'CreateQueue', 'Version': '2012-11-05', 'QueueName': 'test-queue'}, 'url': 'http://localhost:4566/', 'context': {'client_region': 'us-west2', 'client_config': <botocore.config.Config object at 0x7fbcd6650a90>, 'has_streaming_input': False, 'auth_type': None}}
2021-11-15 11:54:05,486 - MainThread - botocore.hooks - DEBUG - Event request-created.sqs.CreateQueue: calling handler <bound method RequestSigner.handler of <botocore.signers.RequestSigner object at 0x7fbcd6650a60>>
2021-11-15 11:54:05,486 - MainThread - botocore.hooks - DEBUG - Event choose-signer.sqs.CreateQueue: calling handler <function set_operation_specific_signer at 0x7fbcd40e3dc0>
2021-11-15 11:54:05,487 - MainThread - botocore.auth - DEBUG - Calculating signature using v4 auth.
2021-11-15 11:54:05,487 - MainThread - botocore.auth - DEBUG - CanonicalRequest:
POST
/

content-type:application/x-www-form-urlencoded; charset=utf-8
host:localhost:4566
x-amz-date:20211115T175405Z

content-type;host;x-amz-date
713dbab07a1453e4e230658235262bfa283caf9c80be4087f8789508a72fcbc4
2021-11-15 11:54:05,487 - MainThread - botocore.auth - DEBUG - StringToSign:
AWS4-HMAC-SHA256
20211115T175405Z
20211115/us-west2/sqs/aws4_request
a07441b00cfe624c2cf2bc8df7a206bc086da829837e2363697cd492247aa28a
2021-11-15 11:54:05,487 - MainThread - botocore.auth - DEBUG - Signature:
4ef5db7de8897cb06638345ca39a16d4d62bad3e39d9ac65df042ae98798898c
2021-11-15 11:54:05,487 - MainThread - botocore.endpoint - DEBUG - Sending http request: <AWSPreparedRequest stream_output=False, method=POST, url=http://localhost:4566/, headers={'Content-Type': b'application/x-www-form-urlencoded; charset=utf-8', 'User-Agent': b'aws-cli/2.2.7 Python/3.8.8 Darwin/20.6.0 exe/x86_64 prompt/off command/sqs.create-queue', 'X-Amz-Date': b'20211115T175405Z', 'Authorization': b'AWS4-HMAC-SHA256 Credential=1:Ck3Tx5ke4rfOXBss0u6R/A==LP4WKbF8pYAMuCn7L8CItw==/20211115/us-west2/sqs/aws4_request, SignedHeaders=content-type;host;x-amz-date, Signature=4ef5db7de8897cb06638345ca39a16d4d62bad3e39d9ac65df042ae98798898c', 'Content-Length': '58'}>
2021-11-15 11:54:05,488 - MainThread - urllib3.connectionpool - DEBUG - Starting new HTTP connection (1): localhost:4566
2021-11-15 11:54:05,530 - MainThread - urllib3.connectionpool - DEBUG - http://localhost:4566 "POST / HTTP/1.1" 500 2
2021-11-15 11:54:05,530 - MainThread - botocore.parsers - DEBUG - Response headers: {'Content-Type': 'text/html; charset=utf-8', 'Content-Length': '2', 'date': 'Mon, 15 Nov 2021 17:54:05 GMT', 'server': 'hypercorn-h11'}
2021-11-15 11:54:05,530 - MainThread - botocore.parsers - DEBUG - Response body:
b'{}'
2021-11-15 11:54:05,531 - MainThread - awscli.clidriver - DEBUG - Exception caught in main()
Traceback (most recent call last):
  File "botocore/parsers.py", line 451, in _parse_xml_string_to_dom
  File "<string>", line None
xml.etree.ElementTree.ParseError: not well-formed (invalid token): line 1, column 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "awscli/clidriver.py", line 459, in main
  File "awscli/clidriver.py", line 594, in __call__
  File "awscli/clidriver.py", line 770, in __call__
  File "awscli/clidriver.py", line 901, in invoke
  File "awscli/clidriver.py", line 913, in _make_client_call
  File "botocore/client.py", line 278, in _api_call
  File "botocore/client.py", line 583, in _make_api_call
  File "botocore/client.py", line 603, in _make_request
  File "botocore/endpoint.py", line 102, in make_request
  File "botocore/endpoint.py", line 134, in _send_request
  File "botocore/endpoint.py", line 166, in _get_response
  File "botocore/endpoint.py", line 217, in _do_get_response
  File "botocore/parsers.py", line 243, in parse
  File "botocore/parsers.py", line 505, in _do_error_parse
  File "botocore/parsers.py", line 454, in _parse_xml_string_to_dom
botocore.parsers.ResponseParserError: Unable to parse response (not well-formed (invalid token): line 1, column 0), invalid XML received. Further retries may succeed:
b'{}'

Unable to parse response (not well-formed (invalid token): line 1, column 0), invalid XML received. Further retries may succeed:
b'{}'
rkottamasu@USCHIRKOTTAMA1 localstack %

**Comment by KrishnanRanjithkumar** [comment]
Hi @ramakot I think i've encountered this issue related to invalid XML received. Could you please run `aws configure` command and check your `Default output format ` is `json`?

**Comment by ramakot** [comment]
yes, when I run aws configure, yes I have given all default values and default output format is "json"

**Comment by ramakrr77** [comment]
WARNING:localstack.utils.server.http2_server: Error in proxy handler for request POST http://localhost:4566/: 'NoneType' object has no attribute 'group' Traceback (most recent call last):
  File "/opt/code/localstack/localstack/utils/server/http2_server.py", line 159, in index
    result = await run_sync(handler, request, data)
  File "/opt/code/localstack/localstack/utils/async_utils.py", line 86, in run_sync
    return await loop.run_in_executor(thread_pool, copy_context().run, func_wrapped)
  File "/usr/lib/python3.7/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 545, in handler
    server_address=parsed_url.netloc,
  File "/opt/code/localstack/localstack/utils/aws/request_context.py", line 206, in modify_and_forward
    method, path, data_bytes=data_bytes, headers=headers, *args, **kwargs
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 330, in modify_and_forward
    method=method, path=path, data=data, headers=headers
  File "/opt/code/localstack/localstack/services/edge.py", line 179, in forward_request
    result = do_forward_request(api, method, path, data, headers, port=port)
  File "/opt/code/localstack/localstack/services/edge.py", line 230, in do_forward_request
    result = do_forward_request_inmem(api, method, path, data, headers, port=port)
  File "/opt/code/localstack/localstack/services/edge.py", line 262, in do_forward_request_inmem
    server_address=server_address,
  File "/opt/code/localstack/localstack/services/generic_proxy.py", line 405, in modify_and_forward
    updated_response = update_listener.return_response(**kwargs)
  File "/opt/code/localstack/localstack/services/sqs/sqs_listener.py", line 439, in return_response
    queue_url = re.match(regex, content_str, re.DOTALL).group(1)
AttributeError: 'NoneType' object has no attribute 'group'
2021-11-29T23:44:23:ERROR:asyncio: Exception in callback _chain_future.<locals>._set_state(<Future finis...ute 'group'")>, <Future at 0x...ttributeError>) at /usr/lib/python3.7/asyncio/futures.py:345
handle: <Handle _chain_future.<locals>._set_state(<Future finis...ute 'group'")>, <Future at 0x...ttributeError>) at /usr/lib/python3.7/asyncio/futures.py:345>
Traceback (most recent call last):
  File "/usr/lib/python3.7/asyncio/events.py", line 88, in _run
    self._context.run(self._callback, *self._args)
  File "/usr/lib/python3.7/asyncio/futures.py", line 347, in _set_state
    _copy_future_state(other, future)
  File "/usr/lib/python3.7/asyncio/futures.py", line 317, in _copy_future_state
    assert not dest.done()
AssertionError

**Comment by eamonnsullivan** [comment]
I'm running into this error now too. Is there a workaround? I'm on Ubuntu 20.04, AWS cli version 2.3.1.

**Comment by thrau** [comment]
hi @eamonnsullivan, we have a new SQS provider in place that can maybe solve the issue, could you give it a try? you can activate it by setting `PROVIDER_OVERRIDE_SQS=asf`

**Comment by eamonnsullivan** [comment]
Thanks for the quick response, @thrau , but I get the same error. The [docs](https://docs.localstack.cloud/aws/sqs/) suggest setting SQS_PROVIDER instead of PROVIDER_OVERRIDE_SQS, so I tried both. I also tried setting it to the different options (now three: `asf`, `moto`, `elasticmq`) that I'm aware of, and got the same error too.

The oddest thing about the error message is the mention of XML. I'm configured for json output, but even tried doubly specifying that with `--output json` to no avail.
